ts/README.md

```diff
@@ -0,0 +1,15 @@
+# Usage
```

Comment:
I haven't heard people saying "ts" as much as they say "tms", so not sure it will be obvious what is stands for. Should we spell it out as "translation_services" or something

Reply:
Sure! I was just using the same naming convention as in the Serge configuration file, but we can easily change this.